src/attribute-editor/row.tsx

```diff
@@ -71,15 +63,19 @@ export const Row = React.memo(
         >
           <InternalColumnLayout className={styles['row-control']} columns={definition.length} __breakpoint={breakpoint}>
             {definition.map(({ info, label, constraintText, errorText, control }, defIndex) => (
-              <Attribute
```

Comment:
Is this `<Attribute />` custom form field obsolete now?

Reply:
It was just using `InternalFormField` with hard-coded `stretch` property. Not useful